Doc/whatsnew/3.11.rst

```diff
@@ -906,6 +906,19 @@ sysconfig
   (Contributed by Miro Hrončok in :issue:`45413`.)
 
 
+.. _whatsnew311-tempfile:
+
+tempfile
+--------
+
+* :class:`~tempfile.SpooledTemporaryFile` now fully implements the methods
```

Comment:
```suggestion
* :class:`~tempfile.SpooledTemporaryFile` objects now fully implements the methods
```
Give a referent to "them" in the previous line

Reply:
This change needs `implements` → `implement` too 🙂

Reply:
Oops, that's what I get for editing at 6 am. I'll fix it on main but it doesn't seem worth bothering Pablo to get it into 3.11.0.